miso-web/src/main/webapp/scripts/box_ajax.js

```diff
@@ -67,7 +67,7 @@ var Box = Box || {
     Box.dialogHeight = Box.boxJSON.rows * 40 + 300;
     Box.scanDialog = Box.ScanDialog(scannerName);
     Box.prepareScannerDialog = Box.PrepareScannerDialog(scannerName);
-    Box.scanDiff = Box.ScanDiff();
+    Box.scanDiff = Box.ScanDiff(scannerName);
```

Comment:
Add a fix-level change to the changelog to note what you're fixing here. "Rescan function was not working after a box scanning error"?